### PR TITLE
Moved gvddk to its own project.  Now in github.com/vmware/virtual-disks

### PR DESCRIPTION
**Pull request by dsu-igeek**
Signed-off-by: Dave Smith-Uchida <dsmithuchida@vmware.com>

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,10 +15,11 @@ jobs:
           path: src/github.com/vmware-tanzu/astrolabe
       - name: Download essential GVDDK libraries
         run: |
-          cd src/github.com/vmware-tanzu/astrolabe/vendor/github.com/vmware/gvddk
+          cd /tmp
           wget --quiet https://gvddk-libs.s3-us-west-1.amazonaws.com/VMware-vix-disklib-7.0.0-15832853.x86_64.tar.gz
-          tar xzf VMware-vix-disklib-7.0.0-15832853.x86_64.tar.gz
-          chmod 644 $(find vmware-vix-disklib-distrib/lib64/ -type f)
+	  cd /usr/local
```
Comment (lintongj) [inline .github/workflows/ci.yml:20]: The TAB needs to be removed

**Comment by lintongj** [comment]
Also, please consolidate commits in this PR into a single one.

**Review by lintongj — APPROVED** [review]
/lgtm

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dsu-igeek) [reply .github/workflows/ci.yml]: Thanks

**Pull request merged**